src/middlewares/validateLanguage.js

```diff
@@ -0,0 +1,16 @@
+import { sendFailedResponse } from '../utils/responseHelpers';
+
+function validateLanguage(req, res, next) {
+  const language = req.headers.language;
+  const languageNames = ['En', 'Ru', 'Hy'];
```

Comment:
Why don't we have language enum? Create `languages` enum in constants and use it